plugin/omero_iviewer/iviewer_settings.py

```diff
@@ -40,6 +41,21 @@
          500,
          int,
          "Page size for ROI pagination."],
+
+    "omero.web.iviewer.color_palette":
+        ["COLOR_PALETTE",
+         [],
+         array,
```

Comment:
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-web/1305/console failed with


```
02:04:02 + omero web start
02:04:09 
02:04:09 741 static files copied to '/home/omero/static/web', 2 post-processed.
02:04:10 Clearing expired sessions. This may take some time... Traceback (most recent call last):
02:04:14   File "manage.py", line 75, in <module>
02:04:14     execute_from_command_line(sys.argv)
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
02:04:14     utility.execute()
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/core/management/__init__.py", line 413, in execute
02:04:14     self.fetch_command(subcommand).run_from_argv(self.argv)
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/core/management/base.py", line 354, in run_from_argv
02:04:14     self.execute(*args, **cmd_options)
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/core/management/base.py", line 393, in execute
02:04:14     self.check()
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/core/management/base.py", line 423, in check
02:04:14     databases=databases,
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/core/checks/registry.py", line 76, in run_checks
02:04:14     new_errors = check(app_configs=app_configs, databases=databases)
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/core/checks/urls.py", line 13, in check_url_config
02:04:14     return check_resolver(resolver)
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/core/checks/urls.py", line 23, in check_resolver
02:04:14     return check_method()
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/urls/resolvers.py", line 416, in check
02:04:14     for pattern in self.url_patterns:
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/utils/functional.py", line 48, in __get__
02:04:14     res = instance.__dict__[self.name] = self.func(instance)
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/urls/resolvers.py", line 602, in url_patterns
02:04:14     patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/utils/functional.py", line 48, in __get__
02:04:14     res = instance.__dict__[self.name] = self.func(instance)
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/django/urls/resolvers.py", line 595, in urlconf_module
02:04:14     return import_module(self.urlconf_name)
02:04:14   File "/usr/lib64/python3.6/importlib/__init__.py", line 126, in import_module
02:04:14     return _bootstrap._gcd_import(name[level:], package, level)
02:04:14   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
02:04:14   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
02:04:14   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
02:04:14   File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
02:04:14   File "<frozen importlib._bootstrap_external>", line 678, in exec_module
02:04:14   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/omeroweb/urls.py", line 99, in <module>
02:04:14     __import__(urlmodule)
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/omero_iviewer/urls.py", line 20, in <module>
02:04:14     from omero_iviewer import views
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/omero_iviewer/views.py", line 44, in <module>
02:04:14     from . import iviewer_settings
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/omero_iviewer/iviewer_settings.py", line 61, in <module>
02:04:14     process_custom_settings(sys.modules[__name__], 'IVIEWER_SETTINGS_MAPPING')
02:04:14   File "/home/omero/workspace/OMERO-web/.venv3/lib64/python3.6/site-packages/omeroweb/settings.py", line 1348, in process_custom_settings
02:04:14     setattr(module, global_name, mapping(global_value))
02:04:14 TypeError: array() argument 1 must be a unicode character, not list
```

Reply:
I don't remember how to build the pypi package to test this (sorry) but it sounds like it's angry my default isn't a string. I just pushed a change with quotes around my default.